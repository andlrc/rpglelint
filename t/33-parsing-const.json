[
   {
      "column":34,
      "file":"t/33-parsing-const.rpgle",
      "line":6,
      "message":"'C_CEND' undeclared",
      "severity":"warning"
   }
]
