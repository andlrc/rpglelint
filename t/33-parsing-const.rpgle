// rpglelint: -Wundefined-reference
/include 33-parsing-const.rpgleinc
dcl-c C_BANG    const('!');
dcl-c C_AND     const('&');

return C_BANG + C_AND + C_CBEG + C_CEND;
