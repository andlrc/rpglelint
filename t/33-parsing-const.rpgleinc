dcl-c C_CBEG    const('{');
