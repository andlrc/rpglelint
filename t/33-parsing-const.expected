t/33-parsing-const.rpgle:6:34: warning: 'C_CEND' undeclared [-Wundefined-reference]
 return C_BANG + C_AND + C_CBEG + C_CEND;
                                  ^
